apps/dashboard/app/views/batch_connect/session_contexts/_path_selector.html.erb

```diff
@@ -43,6 +43,11 @@
         <div>
           <ol id="<%= breadcrumb_id %>" class="breadcrumb breadcrumb-no-delimiter">
           </ol>
+
+          <div class="alert alert-warning" role="alert" id="forbidden-warning">
```

Comment:
This is going to throw off the use of `hide` and `show` APIs because they set the styles, but I feel like this should default to `class=d-none`  because sometimes it shows up on initial load (at least for a second) before the javascript runs to hide it.